src/storage_layer.rs

```diff
@@ -92,6 +92,16 @@ impl Visit for JsonStorage<'_> {
             }
         };
     }
+
+    #[cfg(all(tracing_unstable, feature = "valuable"))]
+    #[cfg_attr(docsrs, doc(cfg(all(tracing_unstable, feature = "valuable"))))]
+    fn record_value(&mut self, field: &Field, value: valuable::Value<'_>) {
+        let serializable = valuable_serde::Serializable::new(value);
+        let json_value = serde_json::to_value(serializable).unwrap_or(serde_json::Value::String(
+            "serde_json Serialization error".to_string(),
+        ));
```

Comment:
I'd prefer to skip the value if serialization fails, rather than use a sentinel value.

Reply:
Sure, makes sense. I'll make the change and push it later. Probably later today.

Reply:
That would work!

Reply:
Done in latest. [Link](https://github.com/fluffysquirrels/tracing-bunyan-formatter/blob/9e8abf540feed97e2da05b52cc096129a1b04ff2/src/storage_layer.rs#L101-L118)